### PR TITLE
Adapt frontend

### PR DESCRIPTION
**Pull request by lucasMesquitaBorges**
# What this PR does?
- Allows any CORS origin (required to frontend request the API)
- Persists `access_token` at RSO Authentication
- Adds endpoint `/accounts/me`
- Adds endpoint `/logout`
- When logged endpoint `/` will return a page with link for `My Account` and `Logout`
- Updates README.md

### CODE REVIEW

File: handlers/httpcontrollers/controller.go [hunk 1]
```diff
@@ -20,8 +20,11 @@ const APP_BASE_URL = "http://local.exampleapp.com:3000"
 const APP_CALLBACK_PATH = "/oauth-callback"
 const AUTHORIZE_URL = RSO_BASE_URI + "/authorize"
 const TOKEN_URL = RSO_BASE_URI + "/token"
+const LOGOUT_URL = RSO_BASE_URI + "/logout"
 const APP_CALLBACK_URL = APP_BASE_URL + APP_CALLBACK_PATH
 
+var tokenResponse *TokenResponse
```
Comment (xelaadryth) [inline handlers/httpcontrollers/controller.go:26]: I think a bit too much login-related business logic is starting to bleed into the "controller". We probably want to split out a class that contains the login page HTML and stores the token into its own package, and the controller can simply route to that class's functions. Can be a separate PR.

**Review by xelaadryth — APPROVED** [review]

File: handlers/httpcontrollers/controller.go (hunk 1 above)
Comment (lucasMesquitaBorges) [reply handlers/httpcontrollers/controller.go:26]: OK! I'll mark it as improvement. Thanks
